CONTRIBUTING.md

```diff
@@ -1,11 +1,85 @@
+
+New design proposals to the API spec should be submitted by opening a
+Github issue with a link to a Google Doc containing the proposal. Proposals
+should contain motivation, use cases, and detailed example of proposed
+changes to API interactions. Collaboration on the design, fleshing out of
+use cases, etc can occur as comment discussions in the Google Doc, as well
+as on our weekly calls.
+
+For changes largely impacting the SB API actors (platforms, service authors,
+etc...), it is recommended to solicit feedback from these actors and leave
+enough time (say 2 weeks) for feedback to be provided, and for the
+potentially received objections/suggestions to be handled.
+
+Once the design has been finalized in the Google Doc, the proposed set of
+changes to the specification should be made available for review. This could
+be done by pointing to a branch in a Github repo with the proposed edits.
+Reviewers, or implementers of the feature, can then easily see the exact
+changes being proposed and comment on them within that branch.
+
+Before a formal PR for the changes is created, the feature needs to be
+validated through a full implementation by at least one platform that is
+easily accessible to reviewers. Note that in many cases this would include
+support for the proposed changes by one or more service brokers
+(potentially samples). The support in a platform does not necessarily mean
+that it has been adopted and part of their "upstream" codebase. Rather, the
+proposed changes simply need to be available for review and testing, perhaps
+as unmerged PRs.
+
+Once support for the change has been implemented, a PR should be sent to
+this repo for the change to the broker API specification. By this point,
+the API interactions should be well understood and there should be no
+technical surprises; we expect the only discussion necessary on PRs to be
+for wordsmithing and formatting. It is expected that during this implementation
+phase there will be edits made to the design and proposed specification
+to accurately represent the current status of the proposal.
+
+## PR/Issue Review Process
+
+All proposals (either Pull Requests or Issues) will follow the process
+described below:
+
+- All proposals must be available for review for no less than one week before
+  they are approved. This will provide each dedicated committer enough time
+  to review the proposal without unnecessarily delaying forward progress.
+  Any non-trivial edit to the proposal (e.g. edits larger than typos) will
+  reset the clock.
+- Any dedicated committer can veto (via a "NOT LGTM" comment in the proposal).
+  The comment must include the reasoning behind the veto.
+  It is then expected that the dedicated committers will discuss the concerns
+  and determine the next step for proposal - either close/reject the proposal
+  or address the concerns raised such that the "NOT LGTM" can be recinded.
+- A proposal requires at least 3 "LGTM" comments from at least 3 different
```

Comment:
I'm nervous that so few approvals allows changes to slip through without enough of the PMC members having context.